hack/spectest-report.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -x
+
+# Prysm spectest directory to start from
+start_directory="/prysmaticLabs/prysm/testing/spectest"
```

Comment:
This doesn't resolve for me. Shouldn't it be the project root instead of absolute `/prysmaticLabs/prysm`?